collectors/metrics/test/integration/setup.sh

```diff
@@ -2,181 +2,182 @@
 # Copyright (c) 2021 Red Hat, Inc.
 # Copyright Contributors to the Open Cluster Management project
 
-if [ "$#" -ne 1 ] ; then
+if [ "$#" -ne 1 ]; then
   echo "Usage: $0 IMAGE" >&2
   exit 1
 fi
 
-echo "=====running kind exploration=====" 
+echo "=====running kind exploration====="
 
 IMAGE_NAME=$1
 echo "IMAGE: " $IMAGE_NAME
 
 DEFAULT_NS="open-cluster-management"
 HUB_KUBECONFIG=$HOME/.kube/kind-config-hub
-WORKDIR=`pwd`
+WORKDIR=$(pwd)
 
 sed_command='sed -i-e -e'
 if [[ "$(uname)" == "Darwin" ]]; then
-	sed_command='sed -i '-e' -e'
+  sed_command='sed -i '-e' -e'
 fi
 
 deploy() {
-    #setup_kubectl_command
-	create_kind_hub
-	deploy_prometheus_operator
-	deploy_observatorium
-	deploy_thanos
-	deploy_metrics_collector $IMAGE_NAME	
+  #setup_kubectl_command
+  create_kind_hub
+  deploy_prometheus_operator
+  deploy_observatorium
+  deploy_thanos
+  deploy_metrics_collector $IMAGE_NAME
 }
 
 setup_kubectl_command() {
-	echo "=====Setup kubectl=====" 
-	# kubectl required for kind
-	echo "Install kubectl from openshift mirror (https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.4.14/openshift-client-mac-4.4.14.tar.gz)" 
-	mv README.md README.md.tmp 
-    if [[ "$(uname)" == "Darwin" ]]; then # then we are on a Mac 
-		curl -LO https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.4.14/openshift-client-mac-4.4.14.tar.gz 
-		tar xzvf openshift-client-mac-4.4.14.tar.gz  # xzf to quiet logs
-		rm openshift-client-mac-4.4.14.tar.gz
-    elif [[ "$(uname)" == "Linux" ]]; then # we are in travis, building in rhel 
-		curl -LO https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.4.14/openshift-client-linux-4.4.14.tar.gz
-		tar xzvf openshift-client-linux-4.4.14.tar.gz  # xzf to quiet logs
-		rm openshift-client-linux-4.4.14.tar.gz
-    fi
-	# this package has a binary, so:
-
-	echo "Current directory"
-	echo $(pwd)
-	mv README.md.tmp README.md 
-	chmod +x ./kubectl
-	if [[ ! -f /usr/local/bin/kubectl ]]; then
-		sudo cp ./kubectl /usr/local/bin/kubectl
-	fi
-	# kubectl are now installed in current dir 
-	echo -n "kubectl version" && kubectl version
+  echo "=====Setup kubectl====="
+  # kubectl required for kind
```

Comment:
this all looks repeated? maybe we can simplify

Reply:
Yup!